application/views/index.php

```diff
@@ -1,3 +1,3 @@
 <a href="<?=site_url('registration')?>">
-	<img src="<?=base_url()?>/assets/images/content.jpg">
+	<img src="<?=base_url()?>/assets/images/content.jpg" width="1100">
```

Comment:
But the containers have absolute width, and our users mainly use laptops, so I think this is a not-so-bad solution to this problem.